inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -157,6 +160,7 @@ class Core::Impl : public ICore {
     ITaskExecutor::Ptr _taskExecutor = nullptr;
 
     mutable std::map<std::string, InferencePlugin> plugins;
+    Core& _core;
```

Comment:
why do we need it?

Reply:
To pass it in CacheManagerContext

Reply:
Ok, let's decide first whether we need `CacheManagerContext` itself.